arbitrum-docs/build-decentralized-apps/how-tos/migrate-nfts-arbitrum.mdx

```diff
@@ -0,0 +1,113 @@
+description: Learn how to migrate your NFT collection to Arbitrum through third-party bridges and DIY bridging solutions.
+author: amarrazza
+content-type: how-to
+reader_audience: Developers who want to migrate their NFT collections to Arbitrum
+---
+
+
+import PublicPreviewBannerPartial from '../../partials/_public-preview-banner-partial.md'; 
+
+<PublicPreviewBannerPartial />
+
+This how-to is for **developers** who want to migrate existing NFT collections from Ethereum mainnet to <a data-quicklook-from="arbitrum-one">Arbitrum One</a> or <a data-quicklook-from="arbitrum-nova">Nova</a>. This guidance applies to NFT collections that support web3 games, decentralized apps (dApps), and other use-cases. If your NFTs are minted and managed by smart contracts on Ethereum mainnet, this how-to is for you.
+
+:::info
+
+If you need to bridge ERC-20 tokens to one of Arbitrum's chains, refer to [How to bridge ERC-20 tokens to Arbitrum](#todo) instead.
+
+Also note that unlike ERC-20 tokens, NFTs cannot be migrated to Arbitrum via [bridge.arbitrum.io](https://bridge.arbitrum.io/).
+
+:::
+
+
+## Why migrate NFTs to Arbitrum?
+
+There are three main reasons to migrate your your project's NFTs to Arbitrum:
+
+ 1. **Low fees**. NFTs hosted by one of Arbitrum's chains can be minted, transfered, and traded at a much lower cost compared to Ethereum mainnet.
+ 2. **Fast transactions**. From the end-user's perspective, transactions on Arbitrum's chains are processed more quickly than they are on Ethereum mainnet. This lets you build NFT-powered experiences that feel just about as fast as traditional apps and games.
+ 3. **Interoperability with the Ethereum ecosystem**. If you're migrating your project's NFTs from other non-Ethereum chains, migration to Arbitrum will let your users freely carry their NFTs across application boundaries within the Ethereum ecosystem.
+
+<!-- todo: when shouldn't you? Risks to call out? -->
+
+## NFT token-types and bridging solutions
+
+The following types of tokens can be used to represent NFTs:
+
+| Token type   | Description                                                             |
+| ------------ | ----------------------------------------------------------------------- |
+| **ERC-721**  | Non-fungible tokens that represent ownership of a unique asset.         |
+| **ERC-1155** | A newer standard that allows for more flexibility in how NFTs are used. |
+
+Both of these token-types can be migrated to Arbitrum One and Nova through the use of **third-party NFT bridging solutions** and **DIY bridging solutions**.
+
+:::caution
+
+Although ERC-20 tokens can technically be used as NFTs[^1], this how-to assumes that your NFTs are one of the above token-types.
+
+:::
+
+
+## Migrate NFTs using third-party NFT bridging solutions
+ 
+This option is ideal if you want to migrate your NFT collection to <a data-quicklook-from="arbitrum-one">Arbitrum One</a> or <a data-quicklook-from="arbitrum-nova">Nova</a> as quickly as possible.
```

Comment:
this isn't necessarily the 'quickest' option. it is the most standard option

Reply:
Do we want to _**recommend**_ this option?

Just pushed a change with some "choose this option if _____" scaffolding; goal is to make it very easy for devs to decide which option to select based on concrete tradeoffs that they need to consider for each option. I think we're _almost_ there...